### PR TITLE
chore(plugin): convert traceid in http response headers to hex format

### PR DESCRIPTION
**Pull request by sabertobihwy**
### Checklist

- [x] The Pull Request has tests
- [x] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

* [Implement ...]

### Issue reference
#10270

Fix #_[issue number]_

### CODE REVIEW

File: kong/plugins/opentelemetry/handler.lua [hunk 1]
```diff
@@ -158,6 +159,7 @@ function OpenTelemetryHandler:header_filter(conf)
       local root_span = ngx.ctx.KONG_SPANS and ngx.ctx.KONG_SPANS[1]
       trace_id = root_span and root_span.trace_id
     end
+    local trace_id = to_hex(trace_id)
```
Comment (gszr) [inline kong/plugins/opentelemetry/handler.lua:162]:
```suggestion
    trace_id = to_hex(trace_id)
```

**Review by gszr — CHANGES_REQUESTED** [review]

**Review by gszr — APPROVED** [review]

**Comment by hbagdi** [comment]
This seems to be a fix rather than a chore. Please follow our contributor guidelines (CONTRIBUTING.md).

**Comment by hbagdi** [comment]
@gszr Ping on above.

**Comment by hbagdi** [comment]
@gszr Poke

**Comment by gszr** [comment]
@hbagdi I have taken a few minutes to check on this; a pending CE to EE merge is in order before this can be cherry-picked.

**Comment by gszr** [comment]
> This seems to be a fix rather than a chore. Please follow our contributor guidelines (CONTRIBUTING.md).

This had been fixed at merge time -- see the merged commit 14a64bf

**Pull request merged**